blocks/library/text/index.js

```diff
@@ -44,6 +47,18 @@ registerBlockType( 'core/text', {
 					/>
 				</BlockControls>
 			),
+			focus && (
+				<InspectorControls key="inspector">
+					<div className="blocks-text__drop-cap" style={ { display: 'flex', justifyContent: 'space-between' } }>
```

Comment:
Why do we give this a `className` but then apply its styles inline? Should we move the styles to `styles.scss`?

I think elsewhere @youknowriad had mentioned creating better patterns for inspector sections with labels. Doesn't seem like it should be the responsibility of the block to make sure its inspector controls are aligned as desired.